harness/determined/tensorboard/base.py

```diff
@@ -92,6 +92,17 @@ def sync(
         selector: Callable[[pathlib.Path], bool] = lambda _: True,
         mangler: Callable[[pathlib.Path, int], pathlib.Path] = lambda p, __: p,
         rank: int = 0,
+    ) -> None:
+        # Only sync a maximum of once per second to play nice with cloud storage request quotas.
+        if time.time() - self.last_sync < 1:
+            return
```

Comment:
do we have a test for this "once a second" sync? it'll be awesome to have one for the sake of reliability.

for example, we could add it as one of e2e tests running on GKE: enable debug logs, run a short experiment which logs metrics a few times per second, grab its logs, filter out the log lines for tensorboard files upload, extract their timestamps and check they are only happening once per second.